app/controllers/Shipping.scala

```diff
@@ -49,30 +49,12 @@ class Shipping(touchpointBackend: TouchpointBackend, commonActions: CommonAction
     }
   }
 
-  def viewDeliveryPaperDigital() = NoCacheAction.async {
-    val segment = "paper-digital"
-    catalog.map { catalog =>
-      index(DeliverySubscriptionProduct(
-        title = "Paper + digital home delivery subscription",
-        description =
-          """|If you live within the M25 you can have your papers delivered by 7am Monday - Saturday and 8.30am on Sunday.
-             |Plus you can start using the daily edition and premium live news app immediately.""".stripMargin,
-        altPackagePath = s"/collection/$segment",
-        options = catalog.delivery.list.toList.filter(_.charges.digipack.isDefined).map(planToOptions).sortBy(_.weeklyPrice).reverse
-      ), segment)
-    }
+  def viewDeliveryPaperDigital() = NoCacheAction {
+    Redirect("/collection/paper-digital")
```

Comment:
As discussed, perhaps it would be better to redirect these to the subs landing page? (If there are any links which push people straight to home delivery we don't want them to end up buying vouchers by mistake)